test/e2e/e2e_suite_test.go

```diff
@@ -22,9 +22,10 @@ import (
 )
 
 var (
-	cfg        *rest.Config
-	c          client.Client
-	kubeClient kubernetes.Interface
+	cfg           *rest.Config
+	c             client.Client
+	kubeClient    kubernetes.Interface
+	ImageRegistry string
```

Comment:
Can we put this in a separate file with a comment about the fact that it's expected to be populated via build flags? Should there be a default value (e.g. `testdata`)?

Reply:
Sure, that seems like a good way to clarify this.

Reply:
For the default value, what would you think about creating a default and then in the Makefile changing it to look like this?

```
FOCUS := $(if $(TEST),-v -focus "$(TEST)")
E2E_FLAGS ?= ""
test-e2e: ginkgo ## Run the e2e tests
	$(GINKGO) --tags $(GO_BUILD_TAGS) $(E2E_FLAGS) -trace -progress $(FOCUS) test/e2e
```

Reply:
That seems pretty good. Then downstream we can override E2E_FLAGS, and that's hopefully flexible enough for other cases that might pop up.

Reply:
Cool, those changes are up now. With that in do you see anything outstanding?